docs/src/styles/docs/toc.scss

```diff
@@ -21,10 +21,12 @@
 .docs-toc-inner {
   position: sticky;
   width: 16rem;
+  height: 80vh;
```

Comment:
Just curious, how did you get `80vh` specifically? Is it a precise height, or is it an under/overestimate?

Reply:
No, I just eyeballed `100vh` or `90vh` not look as good as `80vh`.... Any suggestions?

Reply:
I just did some experiment and found since we have a header on the top, if we use `100vh`, the bottom of the ToC would be outside the actual view port

https://user-images.githubusercontent.com/11983489/153946493-be2c8f0a-80ab-45b2-9789-3847c0db6446.mov

Reply:
What about `calc(100vh - var(--docs-header-height))`?